# a PRAC action role as a key / value pair. 

string role_name
string role_value
KeyValue[] role_values
